src/core/matter/mod.rs

```diff
@@ -22,8 +23,10 @@ impl Matter {
         // this unwrap can stay since we have validated that code is not empty, above
         let first = code.chars().next().unwrap();
 
-        const SMALL_VRZ_DEX: [char; 3] = ['4', '5', '6'];
-        const LARGE_VRZ_DEX: [char; 3] = ['7', '8', '9'];
+        lazy_static! {
+            static ref SMALL_VRZ_DEX: [char; 3] = ['4', '5', '6'];
+            static ref LARGE_VRZ_DEX: [char; 3] = ['7', '8', '9'];
+        }
```

Comment:
I'm actually not sure that lazy_static! helps with things like char arrays. It seems most applicable to things like Vecs. I'll try to figure this out.

Reply:
Yeah I mean I believe an array in rust is compiled into the binary, so why bother with this (or the u32s I added it to below). It would be different if we were assigning the value from a function we called.

These values also don't need to change, so const works.